daphne/src/roles/mod.rs

```diff
@@ -344,7 +345,22 @@ mod test {
             let taskprov_leader_token = BearerToken::from("taskprov_leader_token");
             let taskprov_collector_token = BearerToken::from("taskprov_collector_token");
 
-            let prometheus_registry = prometheus::Registry::new();
```

Comment:
rename this (and the struct field it initializes) to `helper_registry`

Reply:
done